### ISSUE TITLE
Sexual violence

### ISSUE THREAD

**Issue by Floppy** [openpolitics/manifesto#338]
From Something New's policy page, this proposal needs turning into a manifesto addition:

> We live in a country where there are 85,000 rapes a year and over 400,000 sexual assaults. The Office of National Statistics estimates there are 1.2 million victims of domestic violence each year. We are outraged by this and by the lack of interest in making an urgent, fundamental change to the safety of women and children in Britain. We propose an immediate Royal Commission on Sex Crimes with a remit to make proposals in justice, policing and rehabilitation to inhibit the harm of current criminals, in the support available for victims, and in any other area to challenge attitudes that create an environment where sexual assault and domestic violence are allowed to be so widespread.

**Comment by philipjohn** [comment]
Without the "Royal" ;)

**Comment by Floppy** [comment]
indeed :)

**Comment by yellowgopher** [comment]
Within the system we have at present, "Royal" would have to remain wouldn't it?

**Comment by andrewdwilliams** [comment]
Agree with @yellowgopher. Abolishing monarchy is a long-term goal, and unless there is a separate policy to de-royalise Commissions, there is no reason to remove the Royal prefix.

**Comment by philipjohn** [comment]
No, it just needs replacing with Independent. It would be a rather silly contradiction for the manifesto to say on one hand we want to abolish the monarchy and then suggest using that monarchy on the other.
